modules/ROOT/pages/build-app-practices.adoc

```diff
@@ -77,13 +77,14 @@ If you use Anypoint Connector for File (File Connector) and share part of the pa
 ----
 
 === HTTP and HTTPS
-For HTTP and HTTPS, use the following placeholders, depending on the deployment model:
 
-* `${http.port}` +
-When deploying to CloudHub or Hybrid using HTTP (port=8081) or HTTPS (port=8082)
+For HTTP and HTTPS ports in on-premises deployments, MuleSoft recommends using the names for property placeholders that are reserved for CloudHub deployments. 
```

Comment:
MuleSoft recommends using > use

Reply:
I don't think we should suggest that they must use those names. I know it breaks our writing style protocol, but i think our protocol is incorrect in this case.